src/lib/plistToJson.ts

```diff
@@ -0,0 +1,36 @@
+// SPDX-FileCopyrightText: Copyright (c) 2021-2025 Objectionary.com
+// SPDX-License-Identifier: MIT
+
+import * as fs from "fs";
+import * as plist from "plist";
+
+/**
+ * Converts a PLIST (XML) syntax file to JSON format
+ *
+ * @param inputPath - Path to the input PLIST (.tmLanguage or .plist) file
+ * @param outputPath - Path where the converted JSON file will be saved
+ *
+ * @throws {Error} If the input file cannot be read, the PLIST content is invalid,
+ *                 or the output file cannot be written
+ *
+ * @example
+ * ```typescript
+ * // Convert .tmLanguage file to JSON
+ * plistToJson('./syntaxes/example.tmLanguage', './syntaxes/example.tmLanguage.json');
+ *
+ * // Convert .plist file to JSON
+ * plistToJson('./syntaxes/example.plist', './syntaxes/example.json');
+ * ```
+ */
+
+export function plistToJson(inputPath: string, outputPath: string) {
```

Comment:
@svmukhin why can't we just call them `input` and `output`? See: https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html